src/components/Table/components/Row.tsx

```diff
@@ -40,7 +40,6 @@ interface RowProps<R extends Kinded> {
   row: GridDataRow<R>;
   style: GridStyle;
   rowStyles: RowStyles<R> | undefined;
-  sortOn: SortOn;
```

Comment:
Removed this prop b/c we can look it up from the TableState; ideally would like to do more of this, to remove the # of props here, but not for this PR.